### ISSUE TITLE
/sys/fs/bpf not mounted on alpine

### ISSUE THREAD

**Issue by jbalonso** [kairos-io/kairos#194]
In the specific case of installing cilium on a manual-install derived from `kairos-alpine-v1.0.0-k3sv1.24.3+k3s1.iso` (configuration turns on k3s but not edgevpn), this error appears in the pod event log for `cilium-agent`, blocking installation:

```
Warning  Failed     36s               kubelet            Error: failed to generate container "a1698747caae77ac00219af8b3157b1435fb952a0f3c7761d599c9d3f3f6ffd4" spec: failed to generate spec: path "/sys/fs/bpf" is mounted on "/sys/fs/bpf" but it is not a shared mount
```

While I could be more fastidious with providing steps to reproduce, I think there is a basic issue with `/sys/fs/bpf` not being a shared mount, which I suspect would be a general issue for CNIs that want to use eBPF.

**Comment by mudler** [comment]
Thanks @jbalonso for opening the issue.

Did you tried a systemd-based image? or is it just affecting alpine? my guess is that we don't do anything specific for /sys/fs/bpf while probably we should

**Comment by jbalonso** [comment]
I just tried `kairos-opensuse-v1.1.0-k3sv1.24.6+k3s1.iso`, and I was able to install cilium without issue. So, I believe you are correct that kairos needs to do something specific about `/sys/fs/bpf`.

However, I consider it important to note that longhorn (which fusses about shared mounts for `/var/lib/rancher`) _also_ works on an installation derived from this ISO. So, I suspect shared mounts might have a more general demand.
